src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.Development.json

```diff
@@ -37,6 +37,9 @@
       "MinutesBeforeReleaseTimeEnd": 1
     }
   },
+  "FeatureManagement": {
```

Comment:
Does omitting this setting in other environments set it to false by default?

Reply:
Yes, it should be off by default.

I doubt that we would ever want to turn it on in any of the other environments as the idea behind it is that we can get this change into the codebase to unblock further development work (i.e. hierarchies within table result subject meta data, ordering of hierarchies, and the front end support), without breaking what we have currently or avoid a long running branch hanging around that all further work depends on.

I would think that as soon as the front end work is done we will remove this setting as I don't think there's any plan for the front end to support the old non-hierarchical format, at least for subject meta data.

I could have just made this toggle depend on a boolean property of the `LocationOptions` class instead but I was experimenting with this dependency to see what it gives us. I suppose we could make use of it in future if we want to keep it registered. 

There should also be somewhere in the App Service UI in Azure to toggle this setting, although without deploying it I don't know what that looks like.

Reply:
Nice one, thanks